.eslintrc.js

```diff
@@ -12,5 +12,46 @@ module.exports = {
     'ember/no-unnecessary-route-path-option': 0,
     'ember/routes-segments-snake-case': 0,
     'sort-imports': 0,
+    "node/no-extraneous-require": ["error", {
+      "allowModules": [
+        "broccoli-file-creator",
+        "broccoli-merge-trees",
+        "browserslist",
+        "caniuse-db",
+        "dotenv",
+        "postcss-scss"
+      ]
+    }]
   },
+  overrides: [
```

Comment:
A lot of this is imported from https://github.com/ilios/eslint-config-ember/blob/master/index.js#L27. I would vote for changes that are needed here probably be needed in other ilios ember apps and they can just get added to `ilios/eslint-config-ember`

Reply:
i yanked those. turns out we don't need that new rule anyway.